stingray/events.py

```diff
@@ -365,71 +374,60 @@ def join(self, other):
         return ev_new
 
     @staticmethod
-    def read(filename, format_='pickle'):
+    def read(filename, format_="pickle", **kwargs):
         """
-        Read an event list from a file on disk. The file must be either a Python pickle file (not recommended
-        for long-term storage), an HDF5 file, an ASCII or a FITS file. The file can have the following
-        attributes in the data or meta-data:
-
-        * ``time``:  the time stamps of the photon arrivals
-        * ``energy``: the photon energy corresponding to each time stamp
-        * ``mjdref``: a reference time in Modified Julian Date
-        * ``notes``: other possible meta-data
-        * ``gti``: Good Time Intervals
-        * ``pi``: some instruments record energies as "Pulse Invariant", an integer number recorded from
-          the Pulse Height Amplitude
-
-        Ascii files need to be ECSV files with an
-        :class:`astropy.timeseries.TimeSeries` containing time, energy and pi
-        in the data cols and the remaining metadata in the ``meta`` attribute
+        Read a :class:`Lightcurve` object from file.
+
+        Currently supported formats are
+
+        * pickle (not recommended for long-term storage)
+        * hea : LC files from all (well, some) HEASARC-supported missions
```

Comment:
Re. `'hea'`, I didn't want to call the format `fits`, because this indicates a generic `fits` file, like those produced by the `write` method. This is the format of HEASOFT-compatible event lists.

Otherwise, I changed the wording and I hope it is a little more clear